### PR TITLE
Adding a checksum value and commenting out windows build since it is …

### PR DESCRIPTION
**Pull request by jaydeland**
…failing atm

#### Summary
Adding checksum and commenting out windows

#### Ticket Link
https://mattermost.atlassian.net/browse/IS-302?atlOrigin=eyJpIjoiODE4NWZhZmFlMmFiNGFlMGJmM2YwZGY5ODNmYjM3ODciLCJwIjoiaiJ9

Also removed the --sort="v:refname" parameter from the build version process as it is not supported on linux.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -169,7 +169,8 @@ jobs:
       - persist_to_workspace:
           root: .
           paths:
-            - ./build/*.tar.gz
+            - ./build/*.tar
```
[the remark this answers is not in the document]
Comment (metanerd) [reply .circleci/config.yml]: agreed!

File: Makefile [hunk 2]
```diff
@@ -23,16 +24,18 @@ package: vendor check
...
 
-	@echo Build Windows amd64
-	env GOOS=windows GOARCH=amd64 go build -mod=vendor
-	zip build/windows_amd64.zip mmctl.exe
+	# @echo Build Windows amd64
+	# env GOOS=windows GOARCH=amd64 go build -mod=vendor
+	# zip build/windows_amd64.zip mmctl.exe
 
-	rm mmctl mmctl.exe
+	rm mmctl #mmctl.exe
```
[the remark this answers is not in the document]
Comment (metanerd) [reply Makefile]: Looks good! Let's wait with merging for the "fix-windows" PR and then uncomment the windows release.

[the remark this answers is not in the document]
Comment (jaydeland) [reply Makefile]: Fixed

**Pull request merged**